transactionoutbox-core/src/main/java/com/synaos/transactionoutbox/Beta.java

```diff
@@ -20,4 +16,6 @@
 @Target({ElementType.TYPE, ElementType.METHOD, ElementType.CONSTRUCTOR})
 @Retention(RetentionPolicy.RUNTIME)
 @Documented
-public @interface Beta {}
+public @interface Beta {
+
```

Comment:
remove unwanted empty line. Some general comments for all the files:
- unwanted lines and spaces removal
- static imports